z02-parts-of-a-graph.md

```diff
@@ -224,19 +226,18 @@ We'll come across more scales later.
 
 In our example, up top, we have these nice labels and tick marks. This is
 something D3 can do for us. We can build an axis, and apply it to a scale. We
-say, "hey, I want to build an axis that".
+say, "hey, I want to build an axis that labels the plot".
 
-<div class="example-row-2">
+<div class="example-row-1">
```

Comment:
The style of this tutorial is to have code on the left side, result on the right side. What is the goal of the single column layout?